code/datums/research_upgrade_datum.dm

```diff
@@ -111,6 +112,26 @@
 	behavior = RESEARCH_UPGRADE_TIER_5 //adjust this to change amount of nanosplints in a stack, cant be higher than five, go change max_amount in the nanosplint itself, then change it.
 	upgrade_type = ITEM_ACCESSORY_UPGRADE
 
+/datum/research_upgrades/item/flamer_tank
+	name = "Upgraded Incinerator Tank"
+	desc = "An upgraded incinerator tank, with larger capacity and able to handle stronger fuels."
+	value_upgrade = 500
+	clearance_req = 1
+	change_purchase = -100
+	minimum_price = 100
+	item_reference = /obj/item/ammo_magazine/flamer_tank/custom/upgraded
+	behavior = null
```

Comment:
Why null behavior? Shouldn't this be `RESEARCH_UPGRADE_TIER_1` or something?

Reply:
if I do use RESEARCH_UPGRADE_TIER_1 the icon_state when the item gets printed is changed for some reason. Hundson told me to just set it to null

Reply:
So again I ask. Why should this be the first to have a null behavior?

Reply:
because behavior is passed to init of an item, a null _should_ just not pass anything, 


Nevermind Im wrong, null gets passed into init, and in this case, right here:
![image](https://github.com/cmss13-devs/cmss13/assets/48099872/65a1e686-2783-48ea-8b68-e8e731ada52e)
And since its one(aka true) it does that.

harmless here, but I really should have added a check for null  earlier when pr was open now that I think about it

Reply:
As a define now this is acceptable; but in a different PR `behavior` really needs to be handled differently. Its a neat idea to allow specifying an argument into the Initialize; but its too unintuitive and restrictive to assume everything should accept a number 1-5.